dist/index.js

```diff
@@ -6152,6 +6154,12 @@ function convertBody(buffer, headers) {
 	// html4
 	if (!res && str) {
 		res = /<meta[\s]+?http-equiv=(['"])content-type\1[\s]+?content=(['"])(.+?)\2/i.exec(str);
+		if (!res) {
+			res = /<meta[\s]+?content=(['"])(.+?)\1[\s]+?http-equiv=(['"])content-type\3/i.exec(str);
+			if (res) {
+				res.pop(); // drop last quote
+			}
+		}
```

Comment:
Not related to this PR really, but my browser shows me tabs-vs-spaces in github diffs, and this file has a [un]healthy mix of both. Could be nice to add an `.editorconfig` or something that keeps things consistent, perhaps?

Reply:
This change comes from the ce733af528c5b1af0144a167663730fbbec86eb2 commit and was generated by running `yarnpkg run package`. You don't think I should be fixing whitespace in the generated `dist/index.js` file do you?

Reply:
I certainly don't 😉